src/V8JsiRuntime.cpp

```diff
@@ -572,19 +589,22 @@ void V8Runtime::initializeV8() {
 }
 
 V8Runtime::V8Runtime(V8RuntimeArgs &&args) : args_(std::move(args)) {
+  counter_stores_ = std::make_shared<CounterStores>();
+  tls_counter_stores_ = counter_stores_;

```

Comment:
Why do we create them in two places: here in constructor and in the CreateNewIsolate() method? In most cases the CreateNewIsolate will override values created here.

Reply:
Ah .. This was a mistake .. Thanks !